benchmarks/domains/CrabNet_AdvOpt.py

```diff
@@ -0,0 +1,160 @@
+"""Crabnet hyperparameter function with 20 continuous and 3 categorical input. 

```

Comment:
@ritalyu17 just to make sure: What is the state of this PR? Should we already give some feedback/comments or will this still change? And also, can you elaborate why you need these very special constraints that you have? And have you tried modelling these with the constraints available in BayBE?

Reply:
not ready, deliberately set to draft

we've talked in the meeting about the constraint, the lookup can simply handle all small numerical problems affecting the constraints

Reply:
Ok, if you discussed this in the meeting then I'll resolve here.